3_run_rsync/tasks/rsync.yml

```diff
@@ -47,7 +113,7 @@
     delay: 3
 
   - name: "Synchronizing files. This may take a while..."
-    shell: "rsync -aPvvH {{ mig_source_data_location }} -e 'ssh -p 2222 -o StrictHostKeyChecking=no -i {{ mig_dest_ssh_key_remote_location }}' {{ mig_dest_ssh_user }}@{{ mig_dest_service_url }}:{{ mig_dest_data_location }}"
+    shell: "rsync -aPvvH {{ mig_source_data_location }} -e 'ssh -p 2222 -o StrictHostKeyChecking=no -i {{ mig_dest_ssh_key_remote_location }}' {{ mig_dest_ssh_user }}@localhost:{{ mig_dest_data_location }}"
```

Comment:
I think you were saying last night you were seeing issue with the known_hosts files. You might try changing `-o StrictHostKeyChecking=no ` to `-o StrictHostKeyChecking=no  -o GlobalKnownHostsFile=/dev/null -o UserKnownHostsFile=/dev/null`